snippets/javascript/caps-lock-on.md

```diff
@@ -0,0 +1,29 @@
+# Detect Caps Lock Is On
```

Comment:
You made a new branch from your caps lock branch. 
Please fix with git checkout origin/master -- snippets/javascript/caps-lock-on.md 

In the future you should **always** first checkout `main` and `git pull` before starting work on a new branch to prevent this issue.